.binder/requirements.txt

```diff
@@ -9,10 +9,3 @@ pandas
 scikit-image
 scikit-learn
 sympy
-
-# checking https://github.com/jupyter-widgets/ipywidgets/issues/3735
```

Comment:
I've removed these here, and opened this issue: https://github.com/jupyter-widgets/team-compass/issues/20